packages/arb-token-bridge-ui/src/components/common/AddCustomChain.tsx

```diff
@@ -176,8 +176,15 @@ async function mapOrbitConfigToOrbitChain(
     data.coreContracts.rollup,
     getProviderForChainId(data.chainInfo.parentChainId)
   )
-  const confirmPeriodBlocks =
-    (await rollup.confirmPeriodBlocks()).toNumber() ?? 150
+
+  let confirmPeriodBlocks = 150
+
+  try {
```

Comment:
This actually didn't work before if `confirmPeriodBlocks` and would through instead.